### ISSUE TITLE
intro essay image permissions check

### ISSUE THREAD

**Issue by njr2128** [cu-mkp/m-k-manuscript-data#1390]
Permissions need to be checked for all intro essays for release 1:
- PHS essays are done
- all Debuiche-Munoz essays need to be checked:
    - 315
    - 316
    - 314
    - 312
    - 317
    - 332

**Comment by ps2270** [comment]
I checked all Debuiche-Munoz essays. Colin's single-authored essays will not be in the first release.

**Comment by ps2270** [comment]
I also checked Munoz essays

**Comment by njr2128** [comment]
I have updated **media-full-list-alt-text-permissions** to reflect this information and the preliminary checks for permissions.

@ps2270 , confirming:

**For release 2:**
- ann_316_ie_19_Debuiche_Dukes-of-Bethune
- ann_314_ie_19_Debuiche_Bertin
--> These have been removed from the first sheet of **media-full-list-alt-text-permissions** spreadsheet and added to a new tab "release-2" (alt-text is done, but permissions are not)

For release 1 / MVP:
- ann_315_ie_19_Munoz_Mint
--> alt-text is done. Permissions check is done - ADDED NEW MEDIA-CREDIT "TOULOUSE ARCHIVE" FOR @thuchacz TO CHECK
- All of the following have alt-text done and preliminary check of permissions done. Followup is required for permissions credit line by @thuchacz. Not all have been added to media-credit tab as I was not sure how they should be added. Instead, I have highlighted the permissions-image-credit cells (column H) on sheet 1 (metadata) in bright yellow
   - 312
   - 317
   - 332
